### ISSUE TITLE
OntologyNode metadata should be able to be prioritize/enabled in config

### ISSUE THREAD

**Issue by bsharpataz** [clulab/eidos#353]
We are finding that some metadata fields are less useful for a given ontology -- a user should be able to specify which field they want to use for grounding in a given ontology

**Comment by kwalcock** [comment]
Do you mean of the fields coming from the Mention?  The canonical name is not always good?
P.S. You should not be awake.

**Comment by bsharpataz** [comment]
Nope, the fields from the ontology, like in the MESH one the descriptions
are throwing us off, so it’d be nice to be able to ground using only
examples there without having to change the ontology file itself (though
officially we could do that instead)

**Comment by kwalcock** [comment]
It looks like Zheng is onto something in the mesh branch.  The canonical name for a mention uses lemmas.  When we add the descriptions to the ontologies, we do not use lemmas.  POS is used to filter for the good words to use, but lemmas are never collected.  The raw words are used directly to look up a vector.  Doesn't this mean that lots of unlemmatized words will not be found?  Capitalization will not be normalized for one, will it?

**Comment by bsharpataz** [comment]
O, it won’t... ok, so then we should decide if we want to use lemmas or
words, and be consistent.  We should decide in part based on the vectors
we’re using, and what they were trained on.  Unlike stems, lemmas are still
words, so theoretically we could go either way?

**Comment by kwalcock** [comment]
Ah, this is the first and only ontology we've had with both descriptions _and_ examples.  Then furthermore the descriptions are not helpful.  Maybe they should be called something else and ignored.  One search and replace can fix that.
